pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/submission/Reference.kt

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.ocio.processingstatusapi.models.submission
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+
+/**
+ * References within a report.
+  * @property type String?
+ * @property key  String?
+ * @property value  String?
+ */
+@GraphQLDescription("Contains Report references.")
+data class Reference(
```

Comment:
Where is this `Reference` class used?  I think it should be deleted.

Reply:
removed